app/routes/endemics/biosecurity.js

```diff
@@ -17,7 +17,7 @@ const previousPageUrl = (request) => {
   const piHuntDone = session?.piHunt === 'yes'
   const piHuntRecommended = session?.piHuntRecommended === 'yes'
   const piHuntAllAnimals = session?.piHuntAllAnimals === 'yes'
-  const piHuntValid = (isPositive && piHuntDone) || (isNegative && piHuntDone && piHuntRecommended && piHuntAllAnimals)
+  const piHuntValid = (isPositive && piHuntDone && piHuntAllAnimals) || (isNegative && piHuntDone && piHuntRecommended && piHuntAllAnimals)
```

Comment:
We need to put this behind the feature flag.

Flag OFF and only isPositive and piHunt === 'yes' is valid
Flag ON and this is valid

Reply:
Good catch, edited 😃